EIPS/eip-1808.md

```diff
@@ -0,0 +1,203 @@
+---
+eip: 1808
```

Comment:
I'm not sure where 1808 came from, but that seems to be an unrelated EIP. This should be numbered 2245 or 2246.